tests/test_builder.py

```diff
@@ -43,15 +46,15 @@ def test_component_builder_at_index_0(self):
         with open(os.path.join(self.output_dir, 'DCMotor_component_builder_0.mo'), 'w') as file:
             file.write(result)
 
-        with open(os.path.join(self.data_dir, 'DCMotor.mo'), 'r') as f:
+        with open(os.path.join(self.data_dir, test_file), 'r') as f:
             diffs = get_diffs(f.readlines(), result.splitlines(keepends=True))
 
-            self.assertEqual(1, len(diffs['additions']), f'should have 1 addition, has these: {diffs["additions"]}')
-            self.assertEqual(0, len(diffs['deletions']), f'should have 0 deletions, has these: {diffs["deletions"]}')
+            assert 1 == len(diffs['additions']), f'should have 1 addition, has these: {diffs["additions"]}'
```

Comment:
no longer inheriting from unittest TestCase so that we can parametrize the class. Move to pytest method of assertions

Reply:
so sad, I like the `self.assert*` methods :(